src/main/java/com/synopsys/integration/alert/provider/DefaultEmailHandler.java

```diff
@@ -93,4 +101,32 @@ public FieldAccessor updateFieldAccessor(final AggregateMessageContent content,
         }
         return Set.of();
     }
+
+    // FIXME temporary fix for license notifications before we rewrite the way emails are handled in our workflow
+    private Set<String> licenseNotificationCheck(final AggregateMessageContent content, final FieldAccessor fieldAccessor, final boolean projectOwnerOnly) {
+        final Optional<LinkableItem> subTopic = content.getSubTopic();
+        if (subTopic.isEmpty()) {
+            final Boolean filterByProject = fieldAccessor.getBoolean(CommonDistributionConfiguration.KEY_FILTER_BY_PROJECT).orElse(false);
```

Comment:
This isn't for this pull request, but in some major version, we should have filterByProject default to true for email. Otherwise, a mistake could generate lots of unwanted emails.